cmd/aries-agentd/startcmd/start.go

```diff
@@ -116,11 +132,16 @@ func Cmd(server server) (*cobra.Command, error) {
 	if err != nil {
 		return nil, fmt.Errorf("tried to mark DB path flag as required but it was not found: %s", err)
 	}
+	startCmd.Flags().StringP(AgentWebhookHostFlagName, AgentWebhookHostFlagShorthand, "", AgentWebhookHostFlagUsage)
+	err = startCmd.MarkFlagRequired(AgentWebhookHostFlagName)
+	if err != nil {
+		return nil, fmt.Errorf("tried to mark agent webhook host flag as required but it was not found: %s", err)
+	}
 
 	return startCmd, nil
 }
 
-func startAgent(server server, host, inboundHost, dbPath string) error {
+func startAgent(server server, host, inboundHost, dbPath, webhookHost string) error {
```

Comment:
@sudeshrshetty @DRK3 suggest creating a struct - the argument count is getting high.

Reply:
Done.